src/app/routes/stake/stake-list/stake-list-card/stake-list-card.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-stake-list-card',
+  templateUrl: './stake-list-card.component.html',
+  styleUrls: ['./stake-list-card.component.scss']
+})
+export class StakeListCardComponent implements OnInit {
+  @Input() organization: string;
+  @Input() organizationImage: string;
+  @Input() providerDate: string;
+  @Input() stakeAmount: number;
+  @Input() stakeRating: number;
+  @Input() activeServices: number;
+  @Input() numberOfNodes: number;
+  @Input() isAcceptingPatrons: Boolean;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  getStakeRatingText(stakeRating: number = this.stakeRating || 0){
```

Comment:
instead of writing `|| 0` you can just assign default value to stakeRating: `@Input() stakeRating = 0`. Anyway pretty nice idea to keep those methods as a pure functions. 👍